server/api/expense-pool/index.js

```diff
@@ -18,6 +18,14 @@ router.get('/:id', isPoolMember(), controller.getPool);
  */
 router.post('/:id/users', isPoolMember(), controller.addUserToPool);
 
+// ------------ Routes above have updated controllers ----------------
+
+// GET => show all pools
+// Added for testing only
+router.get('/test', controller.getPools);
```

Comment:
I think going forward we are going to want to keep testing routes in its own module and in our `routes.js` file do a conditional check:

```
if (app.get('env') === 'development') {
  app.use(someRouterWithTestingRoutes);
}
```